giftcompare/gifts/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from categories.models import Category
+
+# Create your models here.
+
+class Gift(models.Model):
+    class Meta:
+        db_table = "gift"
+
+    name = models.TextField()
+    description = models.TextField()
+    price = models.DecimalField(max_digits = 6, decimal_places = 2)
+    img =  models.URLField()
+    featured_time = models.DateTimeField()
+    source_url = models.URLField()
+    categories = models.ManyToManyField(Category, 
+                                        db_table= "gift_category",
+                                        related_name= "+",
```

Comment:
why is the related name +?

Reply:
It is to avoid to create a backwards relationship so that category doesn't go back to gifts which would be incorrect based on our design.
Basically, a gift can have multiple categories and we want to see them through the categories collection but while a category can be associated with multiple gifts, it is not desirable to have a collection of all the gifts associated with a category within the category model.

For more details: https://docs.djangoproject.com/en/5.0/topics/db/models/#many-to-many-relationships